src/sentry/api_gateway/proxy.py

```diff
@@ -0,0 +1,54 @@
+"""
+Utilities related to proxying a request to a region silo
```

Comment:
This module doesn't only contain utilities -- it has the actual implementation for the proxying behavior.

Are you anticipating other modules in the `api_gateway` package?